<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriceListTable</class>
 <widget class="QWidget" name="PriceListTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1394</width>
    <height>646</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>971</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QComboBox" name="findingComboBox">
      <item>
       <property name="text">
        <string>Номеру</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Названию</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Цене</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="changingButton">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="changingProductComboBox">
      <item>
       <property name="text">
        <string>Номер</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Навзание</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Цену</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPushButton" name="addingProductButton">
      <property name="text">
       <string>Добавить товар</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="8">
     <widget class="QTableWidget" name="priceListTable"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="findingButton">
      <property name="text">
       <string>Найти по</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sortingButton">
      <property name="text">
       <string>Сортировать по</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="sortingComboBox">
      <item>
       <property name="text">
        <string>Номеру (по возрастанию)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Номеру (по убыванию)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Названию (по возрастанию)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Названию (по убыванию)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Цене (по возрастанию)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Цене (по убыванию)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="deletingButton">
      <property name="text">
       <string>Удалить товар</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
